ompi/mca/mtl/ofi/mtl_ofi.h

```diff
@@ -59,23 +59,25 @@ ompi_mtl_ofi_progress(void)
     int count = 0, i, events_read;
     struct fi_cq_err_entry error = { 0 };
     ompi_mtl_ofi_request_t *ofi_req = NULL;
+    struct fi_cq_tagged_entry wc[ompi_mtl_ofi.ofi_progress_event_count];
+
+    memset(&wc, 0, sizeof(struct fi_cq_tagged_entry) * ompi_mtl_ofi.ofi_progress_event_count);
```

Comment:
isn't this kind of an expensive operation to be doing in a progress function?

Reply:
Yeah, this affects latency particularly just after the inject_threshold when we start doing completions. But it's needed for THREAD_MULTIPLE functionality. The "memset()" is strictly speaking not required by OFI, so I think we can safely remove this (I tested after removing the memset and this seems to work fine). Thoughts?

Reply:
Done. FWIW, I see ~5-6% improvement in latency for message sizes just after inject threshold after removing the memset.